Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/model/ConceptEntry.java

```diff
@@ -0,0 +1,308 @@
+    private String word;
+
+    @Column(name="description")
+    @SearchField(fieldName = SearchFieldNames.DESCRIPTION)
+    @LuceneField(lucenefieldName = LuceneFieldNames.DESCRIPTION, isTokenized = true, isMultiple = false, isSortAllowed = true)
+    private String description;
+
+    @Column(name="pos")
+    @SearchField(fieldName = SearchFieldNames.POS)
+    @LuceneField(lucenefieldName = LuceneFieldNames.POS, isTokenized = true, isMultiple = false, isSortAllowed = true)
+    private String pos;
+
+    @Column(name="concept_list")
+    @SearchField(fieldName = SearchFieldNames.CONCEPT_LIST)
+    @LuceneField(lucenefieldName = LuceneFieldNames.CONCEPT_LIST, isTokenized = false, isMultiple = false, isSortAllowed = true)
+    private String conceptList;
+
+    @Column(name="type_id")
+    @SearchField(fieldName = SearchFieldNames.TYPE_ID)
+    @LuceneField(lucenefieldName = LuceneFieldNames.TYPE_ID, isTokenized = false, isMultiple = false, isSortAllowed = true)
+    private String typeId;
+
+    @Column(name="equal_to")
+    @SearchField(fieldName = SearchFieldNames.EQUAL_TO)
+    @LuceneField(lucenefieldName = LuceneFieldNames.EQUALS_TO, isTokenized = false, isMultiple = true)
+    private String equalTo;
+
+    @Column(name="similar_to")
+    @SearchField(fieldName = SearchFieldNames.SIMILAR_TO)
+    @LuceneField(lucenefieldName = LuceneFieldNames.SIMILAR_TO, isTokenized = false, isMultiple = true)
+    private String similarTo;
+
+    @Column(name="synonym_ids")
+    @SearchField(fieldName=SearchFieldNames.SYNONYM_ID)
+    @LuceneField(lucenefieldName = LuceneFieldNames.SYNONYMID, isTokenized = false, isMultiple = true)
+    private String synonymIds;
+
+    @Column(name="synset_ids")
+    private String synsetIds;
+
+    @Column(name="narrows")
+    private String narrows;
+
+    @Column(name="broadens")
+    private String broadens;
+
+    @Column(name="creator_id")
+    @SearchField(fieldName = SearchFieldNames.CREATOR)
+    @LuceneField(lucenefieldName = LuceneFieldNames.CREATOR, isTokenized = false, isMultiple = false)
+    private String creatorId;
+
+    @Column(name="modified")
+    @SearchField(fieldName = SearchFieldNames.MODIFIED)
+    @LuceneField(lucenefieldName = LuceneFieldNames.MODIFIED, isTokenized = false, isMultiple = false)
+    private String modified;
+
+    @Column(name="merged_ids")
+    @SearchField(fieldName = SearchFieldNames.MERGED_IDS)
+    @LuceneField(lucenefieldName = LuceneFieldNames.MERGED_IDS, isTokenized = false, isMultiple = true)
+    private String mergedIds;
+
+    @Column(name="is_deleted")
+    private boolean isDeleted;
+    
+    @Column(name="modified_user")
+    private String modifiedUser;
+
+    @OneToMany( cascade = CascadeType.ALL, fetch= FetchType.EAGER)
+    @JoinColumn(name = "concept_entry_id", referencedColumnName="concept_id")
+    private List<ChangeEvent> changeEvents = new ArrayList<>();
+     
+    @Column(name="alternative_ids")
+    private String alternativeIds;
```

Comment:
why did you change this to a string? A lot of the code won't work anymore (see comment below).